src/core/checkedint.d

```diff
@@ -741,4 +763,4 @@ unittest
     assert(mulu(0UL, 0UL, overflow) == 0);
     assert(overflow);                   // sticky
 }
-}
```

Comment:
tolf is also used in druntime's posix.mak to canonicalize the line endings.
Other than that, I approve this PR.